### PR TITLE
upgrade cloud-provider-openstack-e2e-test-csi-cinder use golang 1.16

### PR DESCRIPTION
**Pull request by jichenjc**
Fixs: #695

### CODE REVIEW

File: zuul.d/jobs.yaml [hunk 1]
```diff
@@ -849,6 +849,8 @@
     secrets:
       - citynetwork_credentials
       - swr
+    vars:
+      go_version: '1.15'
```
Comment (ramineni) [inline zuul.d/jobs.yaml:853]: @jichenjc Its already enabled. https://github.com/theopenlab/openlab-zuul-jobs/pull/1129/files

Comment (jichenjc) [reply zuul.d/jobs.yaml:853]: @ramineni why it's not running well on our CI?

**Comment by wangxiyuan** [comment]
I assume this can be closed?

**Comment by jichenjc** [comment]
yes

**Pull request closed without merging**